docs/source/cloud-setup/policy.rst

```diff
@@ -0,0 +1,222 @@
+.. _advanced-policy-config:
+
+Admin Policy Enforcement
+========================
+
+
+SkyPilot allows admins to enforce policies on users' SkyPilot usage by applying
+custom validation and mutation logic on user's task and SkyPilot config.
+
+In short, admins offers a Python package with a customized inheritance of SkyPilot's
+``AdminPolicy`` interface, and a user just needs to set the ``admin_policy`` field in
+the SkyPilot config ``~/.sky/config.yaml`` to enforce the policy to all their
+tasks.
```

Comment:
```suggestion
SkyPilot provides an **admin policy** mechanism that admins can use to enforce certain policies on users' SkyPilot usage. An admin policy applies
custom validation and mutation logic to a user's tasks and SkyPilot config.

Example usage:

  - Adds custom labels to all tasks [Link to below, fix case]
  - Always Disable Public IP for AWS Tasks [Link to below]
  - Enforce Autostop for all Tasks [Link to below]
 
To implement and use an admin policy:

- Admins writes a simple Python package with a policy class that implements SkyPilot's
``sky.AdminPolicy`` interface; 
- Admins distributes this package to users;
- Users simply set the ``admin_policy`` field in
the SkyPilot config file ``~/.sky/config.yaml`` for the policy to go into effect.
```

As I write this I realize right now users may still forget to install the policy module. Maybe it's a good idea to add a Tip box: saying in the future, SkyPilot may provide a client-server architecture so that a centralized server can install and enforce such admin policies.